app/src/main/java/com/doctor/yumyum/presentation/ui/write/WriteFragment2.kt

```diff
@@ -18,27 +21,28 @@ import com.doctor.yumyum.databinding.FragmentWriteSecondBinding
 
 class WriteFragment2 : BaseFragment<FragmentWriteSecondBinding>(R.layout.fragment_write_second), View.OnClickListener{
     private lateinit var changeIngredients : ActivityResultLauncher<Intent>
+    private val write2ViewModel: Write2ViewModel by viewModels()
 
     companion object {
         const val REQUEST_CODE_ADD_INGREDIENTS = 9001
         const val REQUEST_CODE_MINUS_INGREDIENTS = 9002
     }
 
-
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
         initBinding()
         initListener()
         changeIngredients()
-
     }
 
     private fun initBinding() {
         binding.secondFragment = this
+        binding.write2ViewModel = write2ViewModel
```

Comment:
다른 변수들이랑 동일하게 `writeSecondViewModel`로 하면 좋을 것 같아용
```suggestion
        binding.writeSecondViewModel = writeSecondViewModel
```